cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/Users/iliamo/ChronoSync.ns3/.waf-1.7.10-5941bd303cc0187a7c1b55a94964b18a/waflib/Tools/c_config.py", line 459, in run_c_code\n    bld.compile()\n  File "/Users/iliamo/ChronoSync.ns3/.waf-1.7.10-5941bd303cc0187a7c1b55a94964b18a/waflib/Build.py", line 190, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 4519721616: cxx test.cpp -> test.cpp.1.o}\n[\'/usr/bin/g++\', \'-O3\', \'-g\', \'-I\', \'-I/opt/local/include\', \'-DNS3_MODULE=1\', \'-DHAVE_SSL=1\', \'-DHAVE_NS3_NDNSIM=1\', \'-DHAVE_NS3_CORE=1\', \'-DHAVE_NS3_NETWORK=1\', \'-DHAVE_NS3_INTERNET=1\', \'-DHAVE_NS3_POINT_TO_POINT=1\', \'../test.cpp\', \'-c\', \'-o\', \'test.cpp.1.o\']\n'
